### DISCUSSION TITLE
How to get keyboard shortcuts work in Univerdoc

### DISCUSSION THREAD

**Discussion by xwiz** [dream-num/univer#2302] [Q&A | 常见问题]
I noticed keyboard shortcuts work in sheets demo but not in docs demo.

Is there a way to enable/get it to work in docs?

**Comment by jikkai** [comment]
Thank you for your interest in Univer Doc. Currently, our primary focus is on the development of Univer Sheet. The development of Univer Doc (and Univer Slide) is still in its early stages. We are planning to enter the sprint phase in Q3, so please stay tuned to our roadmap for further updates. 😀
